crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -708,6 +696,14 @@ impl ArtifactFetcher {
             }
         };
 
+        if matches!(sourcemap.uri, CachedFileUri::Embedded) {
+            self.metrics
+                .record_not_found(SourceFileType::SourceMap, false);
```

Comment:
This records embedded sourcemaps as not needed, right? Might it make more sense to record embedded sourcemaps as found?

Reply:
Hm, but you don’t need to *fetch* it from anywhere.

Reply:
Yeah, that's also true. I think it's fine the way it is.